config/application.rb

```diff
@@ -77,5 +77,10 @@ class Application < Rails::Application
 
     # Bring the log under control
     config.lograge.enabled = true
+
+    # Only send referrer information to ourselves
+    config.action_dispatch.default_headers = {
```

Comment:
Of course I only ever have questions after approving... Sorry, sigh. Does this overwrite the default header – which contains a bunch of things according to the guide (https://guides.rubyonrails.org/configuring.html#configuring-action-dispatch) – with only this? And is that because we don't want the other things?

Reply:
You know, I assumed that it would merge, because that's what Rails usually does, but I think you're right that we need to specify all of them.